bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/knob.js

```diff
@@ -35,6 +35,7 @@ export default () => [
   ]),
   pn('dottedPath', 'Dotted Path', 'Length of dotted path segments (using css stroke-dasharray)'),
   pn('borderWidth', 'Border Width', 'Sets the border width of the slider (px value)'),
+  pn('handleSize', 'Handle Size', 'Sets the size of the slider handle (px value)'),
```

Comment:
```suggestion
  pt('handleSize', 'Handle Size', 'Sets the size of the slider handle (px value)'),
```

handleSize also accepts string values, so let‘s allow text here (see https://roundsliderui.com/document.html#handleSize).